Gemfile

```diff
@@ -23,4 +23,8 @@ group(:dev) do
   gem 'guard'
   gem 'guard-rspec'
   gem 'ruby_gntp'
+  gem 'chefspec'
+  gem 'rubocop'
+  gem 'knife-spork'
+  gem 'kitchen-vagrant'
```

Comment:
yeah. @lamont-granquist is right. `:development` is the magical one and I think that sounds like a good idea to stick to that convention.